InternshipBE/ExaLearn.Dal/Entities/Question.cs

```diff
@@ -8,7 +8,7 @@ public class Question
     {
         public int Id { get; set; }
 
-        public int AudioId { get; set; }
+        public int? AudioId { get; set; }
```

Comment:
?

Reply:
> for what?
Because only listening questions have audio file in.